### PR TITLE
 Add social links on footer

### PR DESCRIPTION
**Pull request by brunoti**
Fixes: #1

### CODE REVIEW

**Comment by izanf** [comment]
@brunoti Good work! But, add the flags "noopener" and "noreferrer" on attribute "rel", please.
cuz: https://developers.google.com/web/tools/lighthouse/audits/noopener

File: src/App.js [hunk 1]
```diff
@@ -19,6 +21,14 @@ function App() {
         <p>* Por favor, se possivel trazer notebook para participar do mini curso.</p>
         <p>** É novo por aqui? Preencha <a href="https://docs.google.com/forms/d/e/1FAIpQLSfoa7A91QVldfTJJz7o6chDN4MxMADtjLuSJ5TlipqI6swL4A/viewform" target="_blank" rel="noopener noreferrer">nossa pesquisa</a> para conhecermos melhor o perfil da comunidade.</p>
       </aside>
+      <footer className="App-footer">
+        <a href="https://t.me/joinchat/H1k6RhYZPseq9vZ1W8A5Eg" target="_blank" className="App-social" >
```
Comment (izanf) [inline src/App.js]: @brunoti Good work! But, add the flags "noopener" and "noreferrer" on attribute "rel", please.
cuz: https://developers.google.com/web/tools/lighthouse/audits/noopener

File: src/App.js [hunk 2]
```diff
@@ -19,6 +21,14 @@ function App() {
...
+          <img src={telegram} alt="telegram" />
+        </a>
+        <a href="https://instagram.com/reactnatal" target="_blank" className="App-social">
```
Comment (izanf) [inline src/App.js]: @brunoti Good work! But, add the flags "noopener" and "noreferrer" on attribute "rel", please.
cuz: https://developers.google.com/web/tools/lighthouse/audits/noopener

Comment (brunoti) [reply src/App.js]: Done!

**Pull request merged**